proto/wippersnapper/ds18x20/v1/ds18x20.proto

```diff
@@ -0,0 +1,51 @@
+// SPDX-FileCopyrightText: 2022 Brent Rubell for Adafruit Industries
+// SPDX-License-Identifier: MIT
+// WipperSnapper's Protobuf communication API for DS18X20 Maxim Temperature ICs
+
+syntax = "proto3";
+
+package wippersnapper.ds18x20.v1;
+import "wippersnapper/i2c/v1/i2c.proto";
+
+/**
+* ds18x20InitRequest represents a request to initialize
+* a DS18X20 Maxim temperature sensor, from the broker.
+* NOTE: This API only supports ONE DS18X20 device PER OneWire bus.
+*/
+message Ds18x20InitRequest {
+  int32  onewire_pin        = 1; /** The desired pin to use as a OneWire bus. */
+  uint32 sensor_period      = 2; /** The desired amount of time between sensor reads. */
+  int32  sensor_resolution  = 3; /** The desired sensor resolution (9, 10, 11, or 12 bits). */
+}
+
+/**
+* ds18x20InitResponse represents a device's response
+* to a ds18x20InitRequest message.
+*/
+message Ds18x20InitResponse {
+  bool is_initialized  = 1; /** True if the 1-wire bus has been initialized successfully, False otherwise. */
+}
```

Comment:
need the pin here so i know which one is reporting in.

same style where it is grayed out until we get a response, then show it clearly if successful, and delete it if not?

Reply:
Will add the pin.

> same style where it is grayed out until we get a response, then show it clearly if successful, and delete it if not?

Yep!